### PR TITLE
Amount Display Style Option in Custom Currency

### LINKED ISSUE #644: Option to have a space between the symbol and amount
When using a 3 character currency "symbols" (just letters), Denaro doesn't put a space in-between this symbol and the amount. So, for example, you would see USD1 rather than the correct USD 1. For this reason, Denaro should allow 4 characters as a symbol, or add an option to have a space between the symbol and amount.

### PR DESCRIPTION
**Pull request by nlogozzo**
Closes #644 

image

### CODE REVIEW

File: NickvisionMoney.Shared/Models/AccountMetadata.cs [hunk 1]
```diff
@@ -121,6 +124,24 @@ internal AccountMetadata(string name, AccountType accountType)
...
+    /// The style to use for displaying an amount
+    /// </summary>
+    /// <remarks>Must be a value between 0 and 4. See https://learn.microsoft.com/en-us/dotnet/api/system.globalization.numberformatinfo.currencypositivepattern?view=net-7.0#remarks for the values' meaning</remarks>
+    public int? CustomCurrencyAmountStyle
+    {
+        get => _customCurrencyAmountStyle;
+
+        set
+        {
+            if (value != null && (value < 0 || value > 4))
```
Comment (fsobolev) [inline NickvisionMoney.Shared/Models/AccountMetadata.cs:137]: `value > 3`?

**Review by fsobolev — CHANGES_REQUESTED** [review]

**Comment by fsobolev** [comment]
`n $` looks okay, but I think it can be improved. We could update ComboRow's model on changing currency symbol and use it, showing "100 🪙" for example.

File: NickvisionMoney.Shared/Models/AccountMetadata.cs (hunk 1 above)
Comment (nlogozzo) [reply NickvisionMoney.Shared/Models/AccountMetadata.cs:137]: Ah yes... could you change that and the comment for me?

**Comment by nlogozzo** [comment]
> `n $` looks okay, but I think it can be improved. We could update ComboRow's model on changing currency symbol and use it, showing "100 🪙" for example.

That would work... world you like to tackle it or want me?

**Comment by fsobolev** [comment]
> That would work... world you like to tackle it or want me?

I'm going to be busy for a while so I guess you will make it faster.

**Comment by nlogozzo** [comment]
image
image

**Review by fsobolev — APPROVED** [review]

**Pull request merged**